debezium-core/src/main/java/io/debezium/transforms/debeziumtimestampcoverter/DebeziumTimestampConverter.java

```diff
@@ -0,0 +1,698 @@
+                        .version(1).optional().build().schema();
+            }
+        });
+
+        TRANSLATORS.put(TYPE_EPCOH_DATE, new TimestampTranslator() {
+            @Override
+            public Date toRaw(Config config, Object orig) {
+                return Date.from(java.time.LocalDate.ofEpochDay((Integer) orig).atStartOfDay(ZoneId.systemDefault()).toInstant());
+            }
+
+            @Override
+            public Schema typeSchema() {
+                return org.apache.kafka.connect.data.Date.SCHEMA;
+            }
+
+            @Override
+            public Object toType(Config config, Date orig, String format) {
+                Calendar origCalendar = Calendar.getInstance(UTC);
+                origCalendar.setTime(orig);
+                Calendar result = Calendar.getInstance(UTC);
+                result.setTimeInMillis(0L);
+                result.set(Calendar.HOUR_OF_DAY, origCalendar.get(Calendar.HOUR_OF_DAY));
+                result.set(Calendar.MINUTE, origCalendar.get(Calendar.MINUTE));
+                result.set(Calendar.SECOND, origCalendar.get(Calendar.SECOND));
+                result.set(Calendar.MILLISECOND, origCalendar.get(Calendar.MILLISECOND));
+                return result.getTime();
+            }
+
+            @Override
+            public Schema optionalSchema() {
+                return SchemaBuilder.int32()
+                        .name(org.apache.kafka.connect.data.Date.LOGICAL_NAME)
+                        .version(1).optional().build().schema();
+            }
+        });
+
+        OPTIONAL_TRANSLATORS.put(TYPE_INT64, () -> {
+            return Schema.OPTIONAL_INT64_SCHEMA.schema();
+        });
+
+        OPTIONAL_TRANSLATORS.put(TYPE_INT32, () -> {
+            return Schema.OPTIONAL_INT32_SCHEMA.schema();
+        });
+
+        OPTIONAL_TRANSLATORS.put(TYPE_INT16, () -> {
+            return Schema.OPTIONAL_INT16_SCHEMA.schema();
+        });
+
+        OPTIONAL_TRANSLATORS.put(TYPE_INT8, () -> {
+            return Schema.OPTIONAL_INT8_SCHEMA.schema();
+        });
+
+        OPTIONAL_TRANSLATORS.put(TYPE_FLOAT64, () -> {
+            return Schema.OPTIONAL_FLOAT64_SCHEMA.schema();
+        });
+
+        OPTIONAL_TRANSLATORS.put(TYPE_FLOAT32, () -> {
+            return Schema.OPTIONAL_FLOAT32_SCHEMA.schema();
+        });
+
+        OPTIONAL_TRANSLATORS.put(TYPE_BOOLEAN, () -> {
+            return Schema.OPTIONAL_BOOLEAN_SCHEMA.schema();
+        });
+
+        OPTIONAL_TRANSLATORS.put(STRING_TYPE, () -> {
+            return Schema.OPTIONAL_STRING_SCHEMA.schema();
+        });
+
+        OPTIONAL_TRANSLATORS.put(TYPE_BYTES, () -> {
+            return Schema.OPTIONAL_BYTES_SCHEMA.schema();
+        });
+
+        OPTIONAL_TRANSLATORS.put(TYPE_STRUCT, () -> {
+            return SchemaBuilder.type(Schema.Type.STRUCT).optional().build().schema();
+        });
+
+    }
+
+
+    public static Boolean containsField(String[] field_type, String field_name) {
```

Comment:
This doesn't need to be public, but can be `protected` if you still want to use it in a unit test in the same package (would be good to do).